inception-project/inception
# #950 - Add LAPPS Grid external recommender support

**What's in the PR**
* Add new module for LAPPS grid recommender integration

**How to test manually**
* Create a new recommender for NER or POS layer
* Make sure you get recommendations
* Make sure that you can create LAPPS grid recommender just for NER and POS

**Automatic testing**

**Documentation**

**Todos**
* [ ] Curate list of recommenders that work
* [ ] Write integration test
